data/governments.txt

```diff
@@ -1017,25 +1020,106 @@ government "Pug (Wanderer)"
 	"attitude toward"
 		"Drak" -.01
 		"Quarg" -.01
+		"Quarg (Hai)" -.01
+		"Quarg (Kor Efret)" -.01
+		"Quarg (Gegno)" -.01
 	"friendly hail" "friendly pug"
 	"friendly disabled hail" "friendly disabled pug"
 	"hostile hail" "hostile pug"
 	"hostile disabled hail" "hostile pug"
 
+# Quarg are split up to their respective areas, and an overall one is kept for missions or other use.
 government "Quarg"
 	swizzle 0
 	color .88 .77 0
 	
 	"player reputation" 1
 	"attitude toward"
+		"Quarg (Hai)" 1
+		"Quarg (Kor Efret)" 1
+		"Quarg (Gegno)" 1
 		"Merchant" .01
-		"Kor Efret" .01
-		"Kor Mereti" -.01
-		"Kor Sestor" -.01
 		"Hai" .01
```

Comment:
Why are you removing these?

Reply:
Do the human Quarg ever come into contact with the Automata? I forget.

Reply:
This is under the `Quarg` government, and that's the government of every existing Quarg ship, including ones created in missions that are specifically intended to fight the automata (the ships they send to help the Wanderers defend Kaliptari, the Quarg ships that assist in the mission where you defend the Efreti from a Mereti attack, and the Quarg that assist in the defence of Alnitak, off the top of my head).

Even if you update those missions to use the sub-governments, this government still needs to maintain continuity for the ships that already exist.

Reply:
By ships that already exist, do you mean fleets, or ones active in missions? I also updated systems with fleets that use the specific governments. But yeah, good point.